### DISCUSSION TITLE
Pesquisa sobre o processo seletivo perfeito (ou quase)

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#1276] [Discussão]
Galera, estou fazendo mais uma pesquisa sobre o mercado de trabalho. Agora para dar mais voz a nós, devs e devas, sobre o processo seletivo das empresas.

As pessoas falam de experiência de compra, experiência do cliente, mas às vezes esquecem da employee experience, ou experiência do empregado/funcionário, para reter ou contratar pessoas.

O processo de contratação é a porta de entrada para a nossa experiência dentro da empresa e se ele for ruim a gente não vai mais querer entrar lá e até não indicar ela para mais ninguém.

Visando levar essa informação pra cima (liderança, gestão e diretorias de empresas) estou puxando essa pesquisa.

Responda o formulário para o mercado saber o que você acha que é um bom processo seletivo. ;)

O resultado será postado no [meu blog](https://woliveiras.com.br/pesquisas/) (que inclusive tem outras pesquisas ativas) assim que obtivermos um bom número de respostas.

**Formulário:** [Pesquisa sobre o processo seletivo perfeito (ou quase)](http://bit.ly/pd-processo-seletivo)

**Comment by Roger-Melo** [comment]
respondido! \o/

**Comment by deppbrazil** [comment]
@woliveiras fechou a pesquisa? onde foi postado os resultados?
